### DISCUSSION TITLE
skip-tests in helmDefaults, so helmfile template does not render tests?

### DISCUSSION THREAD

**Discussion by reegnz** [helmfile/helmfile#671] [Ideas]
helm by default renders tests if running `helm template`. If you add the `--skip-tests` flag it allows the tests not to be rendered with `helm template`.

Helmfile passes along the `--skip-tests` flag properly, but it would be nice to be able to also skip tests declaratively, either by:

a, always skipping rendering tests when `helmfile template` is run OR
b, allow a `skipTests` flag in `helmDefaults` in helmfile.yaml

Currently I have to invoke helmfile with `helmfile template --skip-tests` and it would be nice not having to explicitly define it.

**Comment by reegnz** [comment]
I think the best option would be to add these as first-class citizens into the `helmDefaults`:
- skipTests
- skipHooks
- skipCrds

**Reply by yxxhero** [reply]
@reegnz PR is welcome. Thanks very much.

**Reply by reegnz** [reply]
Cool, will look into it then.
